notebooks/samples/pytorch/lightning/TrainingAndPredictionWithPyTorchLightning.ipynb

```diff
@@ -85,7 +85,7 @@
       "outputs": [],
```

Comment:
<p>Could you pull out these code comments into a text cell above it? That will make it easier to read if a user is running this section. Otherwise, they just skip to the next notebook section as you've indicated in the second paragraph.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/244/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>